src/main.ts

```diff
@@ -59,7 +59,7 @@ export async function run(): Promise<void> {
     }
     process.exit(0)
   } catch (error) {
-    log.fail("unknown", error.message)
+    log.fail("unknown", (error as Error).message)
```

Comment:
Is this guaranteed to be of type `Error`? If it's not then `if (error instanceof Error)...` will be safer

Reply:
Yes, but what do we do in the else case?

I was thinking it could be better to let this crash hard so we know what to do else.

Reply:
I don't think it'll crash hard since it's Javascript, instead it'll probably be the unhelpful `undefined`.
You could `log.fail("unknown", error)` instead so you get the default representation?

Reply:
Typescript though - how do I do it without getting type errors etc? =/

I pushed a commit making a check